oggmev/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: "OGGM e.V."
+description: "The OGGM organization"
+header-img: "img/backgrounds/dahlbreen_v1.jpg"
+---
+
+**OGGM e.V.** is a non-profit organization based in Bremen, Germany.
+
+The purpose of the organization is the promotion of science and research
+in the field of climate and glaciology, in particular by promoting
+the development of an open source model for glaciers dynamics. This implies:
+- the organization and participation to events (conferences, workshops,
+  seminars...) in relation to climate and cryospheric sciences
+- the publication of scientific or vulgarization material in peer-review
```

Comment:
what is meant by "vulgarization"?

Reply:
Sorry, this is french for "popular science" (https://en.wikipedia.org/wiki/Popular_science). What about:

```
the publication of scientific or popular science material in ...
```

?

Reply:
perhaps then: 
the preparation of technical and popular material for publication in peer-reviewed journals, newspapers, websites, social media, etc.